pkg/neonvm/controllers/vm_controller.go

```diff
@@ -876,14 +876,10 @@ func updatePodMetadataIfNecessary(ctx context.Context, c client.Client, vm *vmv1
 		ignoreExtra map[string]bool // use bool here so `if ignoreExtra[key] { ... }` works
 	}{
 		{
-			metaField: "labels",
-			expected:  labelsForVirtualMachine(vm, nil), // don't include runner version
-			actual:    runnerPod.Labels,
-			ignoreExtra: map[string]bool{
-				// Don't override the runner pod version - we need to keep it around without
-				// changing it; otherwise it's not useful!
-				vmv1.RunnerPodVersionLabel: true,
-			},
+			metaField:   "labels",
+			expected:    labelsForVirtualMachine(vm), // don't include runner version
```

Comment:
comment is outdated now, with the changes we don't have runner version argument at all

Reply:
good catch, thanks! Fixed in a6d39ee32c7171aff2a70efbd865bc183526289e